custom_components/ha_washdata/ml/training_task.py

```
@@ -0,0 +1,605 @@
+        return importlib.import_module(f"{__package__}.{module_name}")
+    except Exception:  # pylint: disable=broad-exception-caught
+        return None
+
+
+def _baseline_auc(capability: str, X_test: np.ndarray, y_test: np.ndarray, columns: list[str]) -> float | None:
+    module = _embedded_module(capability)
+    if module is None:
+        return None
+    try:
+        scores = np.array(
+            [float(module.score(dict(zip(columns, row)))) for row in X_test], dtype=float
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)